docs/getting_started/basic_usage/basic_usage.rst

```diff
@@ -27,58 +27,26 @@ as well as some training parameters.
 Some common config parameters:
 
 task
-    The task, such as ``train`` or ``forward``.
+    The task, such as ``train``, ``forward`` or ``search``.
 
 device
     E.g. ``gpu`` or ``cpu``. Can also be ``gpu0,gpu1`` for multi-GPU training.
 
 use_tensorflow
     If you set this to ``True``, TensorFlow will be used.
 
-train / dev
+train / dev / eval
     The datasets. This can be a filename to a hdf-file.
     Or it can be a dict with an entry ``class`` where you can choose a from a variety
     of other dataset implementations, including many synthetic generated data.
+    ``train`` and ``dev`` are used during training, while ``eval`` is usually used to define the dataset for the
+    ``forward`` or ``search`` task.
 
-extern_data (former num_outputs)
+extern_data
+    A dictionary
```

Comment:
Shouldn't there be a dot at the end?

Reply:
This seems to be wrong in general, this text is incomplete